crates/tx-relay/src/evm/vanchor.rs

```diff
@@ -122,8 +122,16 @@ pub async fn handle_vanchor_relay_tx<'a>(
             .map(|v| v.to_fixed_bytes().into())
             .collect(),
         output_commitments: [
-            cmd.proof_data.output_commitments[0].to_fixed_bytes().into(),
-            cmd.proof_data.output_commitments[1].to_fixed_bytes().into(),
+            cmd.proof_data
+                .output_commitments
+                .get(0)
+                .map(|c| c.to_fixed_bytes().into())
+                .unwrap_or_default(),
+            cmd.proof_data
+                .output_commitments
+                .get(1)
+                .map(|c| c.to_fixed_bytes().into())
+                .unwrap_or_default(),
```

Comment:
In case of missing values these should return default U256 with all zeros, which will fail the signature check.

Reply:
Instead of doing that, could we just map all the `cmd.proof_data.output_commitments` into the expected output commitments as bytes? Like:

```suggestion
                cmd.proof_data
                .output_commitments
                .into_iter()
                .map(|c| c.to_fixed_bytes().into())
                .collect()
```

That would be useful in the future to also support more than two commitments.

Reply:
Had to adjust it a bit but it works.